src/m365/teams/commands/chat/chat-message-list.ts

```diff
@@ -0,0 +1,69 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import Utils from '../../../../Utils';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { Message } from '../../Message';
```

Comment:
Rather than using our custom `Message` type, which we're aiming to remove, let's use the type from the MS Graph types packages (we need `ItemBody` for a cast in line 40):

```ts
import { ItemBody, Message } from '@microsoft/microsoft-graph-types';
```

Reply:
Ok, I used `ItemBody` from the graph-types